guide/14-deep-learning/address_standerdization_using_model_extension.ipynb

```diff
@@ -0,0 +1,955 @@
+{
```

Comment:
<p>Other keys are the prerogative of the Model extension author.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Completed</p>